lib/foreman_maintain/concerns/firewall/maintenance_mode.rb

```diff
@@ -0,0 +1,36 @@
+module ForemanMaintain
+  module Concerns
+    module Firewall
+      module MaintenanceMode
+        def notify_and_ask_to_install_firewall_utility
+          puts 'Unable to find nftables or iptables!'
+          question, pkg = question_and_pkg_name
+          answer = ask_decision(question, actions_msg: 'y(yes), q(quit)')
+          if answer == :yes
+            packages_action(:install, pkg)
+            feature(:instance).firewall.enable_maintenance_mode
+          end
+        end
+
+        def can_install_nft?
+          # The nftables is default from EL8 and Debian 10(Buster)
+          el_major_version >= 8 || deb_major_version >= 10
```

Comment:
What about Ubuntu? If my memory is correct, they do have nftables by default since 21.10 (or rather 22.04 LTS, as that's what we would support in the future)